queue_services/entity-filer/flags.json

```diff
@@ -1,5 +1,6 @@
 {
     "flagValues": {
-        "enable-involuntary-dissolution": true
+        "enable-involuntary-dissolution": true,
+        "namex-nro-decommissioned": true
```

Comment:
Are these the default FF values? If so, wouldn't it better to disable the feature if LaunchDarkly can't be reached?

Reply:
nope. only in local (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/flags.py#L71)